src/main/java/io/cdap/plugin/gcp/bigquery/source/BigQueryAvroToStructuredTransformer.java

```diff
@@ -92,8 +91,7 @@ protected Object convertField(Object field, Schema.Field schemaField) throws IOE
               LocalDateTime.parse(field.toString());
             } catch (DateTimeParseException exception) {
               throw new UnexpectedFormatException(
-                String.format("Field '%s' of type '%s' with value '%s' is not in ISO-8601 format.",
-                              schemaField.getName(),
+                String.format("Datetime field with value '%s' is not in ISO-8601 format.",
```

Comment:
This means the field name is no longer in the error message? Or does this exception get caught somewhere else and rethrown with the field name added?

Reply:
Gets caught in https://github.com/cdapio/hydrator-plugins/blob/develop/hydrator-common/src/main/java/io/cdap/plugin/common/RecordConverter.java#L99 and field name gets added there.